spec/parallel_tests/tasks_spec.rb

```diff
@@ -29,6 +29,42 @@
     end
   end
 
+  describe '.purge_before_load?' do
+    subject { ParallelTests::Tasks.purge_before_load? }
+    context "PURGE_BEFORE_LOAD is set to 'false'" do
```

Comment:
FYI I'd test this like so: (less nesting / context switching and 1 change per test)

```Ruby
it "is true" do
  exoect(subject).to eq true
end

it "is false when set to false" do
  with_env(PURGE_BEFORE_LOAD: 'false') do
    expect(subject).to eq false
  end
end

it "is true when set to true" do
  with_env(PURGE_BEFORE_LOAD: 'true') do
    expect(subject).to eq true
  end
end
```